_data/finance.yml

```
@@ -70,6 +70,7 @@ websites:
       tfa: Yes
       sms: Yes
       phone: Yes
+      doc: https://help.creditkarma.com/hc/en-us/articles/360001472366-Two-Factor-Authentication-FAQ
```

```suggestion
      doc: https://help.creditkarma.com/hc/en-us/articles/360001472366
```
URL can be shortened, which is what we like here ;-)
// Kai